roles/contiv_auth_proxy/tasks/cleanup.yml

```diff
@@ -0,0 +1,10 @@
+---
+
+- name: stop auth-proxy container
+  service: name=auth-proxy state=stopped
+
+- name: cleanup iptables for auth proxy
+  shell: iptables -D INPUT -p tcp --dport {{ item }} -j ACCEPT -m comment --comment "{{ auth_proxy_rule_comment }} ({{ item }})"
```

Comment:
@srampal I would think this could be `command` vs `shell`.  Not sure if the `os_firewall` module can be used here or not.